schwab_api/schwab.py

```diff
@@ -385,6 +387,198 @@ def trade_v2(self,
 
         return messages, False
 
+
+    def option_trade_v2(self,
+        strategy,
+        symbols,
+        instructions,
+        quantities,
+        account_id,
+        order_type,
+        dry_run=True,
+        duration=48,
+        limit_price=0,
+        stop_price=0,
+        valid_return_codes = {0,10},
+        affirm_order=False
+        ):
+        """
+            Disclaimer:
+            Use at own risk.
+            Make sure you understand what you are doing when trading options and using this function.
+            Option trading requires an application and approval process at Schwab.
+            
+            strategy (int) - Type of options strategy:
+                        2 leg strategies:
+                        201 - vertical call spread
+                        202 - vertical put spread
+                        203 - calendar call spread (level 3)
+                        204 - calendar put spread (level 3)
+                        205 - diagonal call spread 
+                        206 - diagonal put spread
+                        207 - ratio call spread (level 3)
+                        208 - ratio put spread (level 3)
+                        217 - custom 2 legs:
+                        3 leg strategies:
+                        209 - Butterfly call spread
+                        210 - Butterfly put spread
+                        218 - custom 3 legs
+                        4 leg strategies:
+                        211 - condor call spread
+                        212 - condor put spread
+                        214 - iron condor
+                        219 - custom 4 legs
+                        Combinations:
+                        226 - straddle
+                        227 - strangle
```

Comment:
I wonder if we might want to consider turning these into a class-wide enumerator; though we can push that to a later CL.

Reply:
I thought the same thing. Might make sense for trades as well like market order, limit order etc

Reply:
But I wanted to get a working version out first and it can always be refined.